lib/video_info/providers/vimeo.rb

```diff
@@ -26,17 +50,34 @@ def embed_url
       end
 
       def date
-        Time.parse(_video['upload_date'], Time.now.utc).utc
+        Time.parse(_video['created_time'], Time.now.utc).utc
       end
 
       def view_count
-        _video['stats_number_of_plays'].to_i
+        _video['stats']["plays"].to_i
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.